### PR TITLE
Fix typos in 09_timeseries.rst

### PR DESCRIPTION
**Pull request by rogererens**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: doc/source/getting_started/intro_tutorials/09_timeseries.rst [hunk 1]
```diff
@@ -382,7 +382,7 @@ More details on the power of time series ``resampling`` is provided in the user
     <div class="d-flex flex-row gs-torefguide">
         <span class="badge badge-info">To user guide</span>
 
-A full overview on time series is given in the pages on :ref:`time series and date functionality <timeseries>`.
+A full overview on time series is given on the pages on :ref:`time series and date functionality <timeseries>`.
```
Comment (jbrockmendel) [inline doc/source/getting_started/intro_tutorials/09_timeseries.rst:385]: i think this one is correct as "in"

**Comment by jbrockmendel** [comment]
one comment, otherwise LGTM pending green

File: doc/source/getting_started/intro_tutorials/09_timeseries.rst (hunk 1 above)
Comment (AlexKirko) [reply doc/source/getting_started/intro_tutorials/09_timeseries.rst:385]: As far as I know, "on" is more correct here grammatically, but it sounds clunky, and we can avoid it. I would rephrase the whole thing like so:
For a full overview on time series, see :ref:`time series section <timeseries>` in the user guide.

Comment (AlexKirko) [reply doc/source/getting_started/intro_tutorials/09_timeseries.rst:385]: As far as I know, "on" is more correct here grammatically, but it sounds clunky, and we can avoid it. I would rephrase the whole thing like so:
For a full overview on time series, see :ref:`time series section <timeseries>` in the user guide.

Comment (jbrockmendel) [reply doc/source/getting_started/intro_tutorials/09_timeseries.rst:385]: cc @WillAyd tie breaker?

Comment (WillAyd) [reply doc/source/getting_started/intro_tutorials/09_timeseries.rst:385]: in is maybe more correct, but I don't think worth changing if its the only thing here. @AlexKirko if you want to reformat sentence separately would definitely be welcome

**Review by WillAyd — APPROVED** [review]

**Comment by WillAyd** [comment]
Thanks @rogererens
